application/controllers/collection_instrument.py

```diff
@@ -419,36 +398,6 @@ def delete_collection_instruments_by_exercise(self, ce_id: str, session: Session
 
         return COLLECTION_EXERCISE_AND_ASSOCIATED_FILES_DELETED, 200
 
-    @staticmethod
-    def publish_remove_collection_instrument(exercise_id, instrument_id):
```

Comment:
Couple of issues here, 1. This shouldn't really be part of this class and has led to the issue with calls being made to external service during sessions. 2. The Collection Exercise is quite dum and cares nothing about the action or the instrument, all it is doing is running a transitionScheduleCollectionExerciseToReadyToReview, which it just needs the exercise id. With that in mind we don't need a remove or an add, or to send it the exercise or action.